Euclidean traveling salesperson.md

```diff
@@ -0,0 +1,40 @@
+# Summary
+This task involves finding the shortest path among a number of “cities” on a map to mimic complex problem solving in which each aspect of a problem depends on other aspects of that problem.
+
+# References
+> Main paper: https://www.pnas.org/content/pnas/115/35/8734.full.pdf
+> > Supporting paper: https://www.pnas.org/content/pnas/suppl/2018/08/09/1802407115.DCSupplemental/pnas.1802407115.sapp.pdf 
+
+# Stimuli
+## The visual components
+> An example TSP from the experiment with the optimal solution
+filled in. To the left is a timer (showing 32s remaining) along with the reset and submit buttons.
+>
+![TSP](images/TSP.png)
+
+# Procedure
+## Steps
+> Obtain informed consent
+> Each subject took a nine-problem pretest, in part to train them on the TSP and in part to assess their individual abilities with respect to solving TSPs
+> Phase One:
+> > Subjects were asked to click with the computer’s mouse on the city icons in the sequence corresponding to the path they wished to submit as their answer. The computer program would draw a line segment connecting each pair of cities as soon as the second city in each leg of the journey had been clicked on.
+> > Each map consisted of 25 cities and thus required 25 separate legs of the overall journey to complete. Participants complete 6 different maps. 
+> > They had 17 tries per map, and 50 seconds per map.
+> Phase Two:
+> > Subjects were put into groups of three, and assigned to one of three conditions.
```

Comment:
Subjects were randomly matched with and anonymous to each other; the order of treatments was also randomly assigned and subjects were exposed to multiple treatments